sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -1689,7 +1689,8 @@
               "dir": "Current Year Income - Supporting Documents",
               "customClass": "font-weight-bold",
               "validate": {
-                "required": true
+                "required": false,
+                "custom": "const maxUploadedFiles = 25; const files = data[instance.key] || []; valid = !files.length ? 'At least one file is required' : files.length > maxUploadedFiles ? `Maximum ${maxUploadedFiles} files allowed` : true;"
```

Comment:
Asking for my understanding. What happens when we enable the required and also have custom validation in place? 

I am asking this because, removing the `required` removes the `*` in the component.
![image](https://github.com/user-attachments/assets/34e93b61-f939-4df7-883a-dde925246188)

Reply:
Is business ok with `*` not being there for mandatory field?

Reply:
Business is ok to have one message for both "required" and custom then I added a `customMessage` and both asterisk and max number of files can be in place.